keel-sql/src/test/kotlin/com/netflix/spinnaker/keel/tx/DeliveryConfigTransactionTests.kt

```diff
@@ -177,11 +156,6 @@ internal class DeliveryConfigTransactionTests : JUnit5Minutests {
         expectThat(resourceRepository.allResourceNames())
           .isEmpty()
       }
-
```

Comment:
This is also no longer an assumption: if the delivery config fails the artifact still might be registered. That's okay.